### ISSUE TITLE
Cannot start the deamon

### ISSUE THREAD

**Issue by alauzon** [wyattjoh/ydns-updater#3]
Hi, thank you for the ydns-updater tool.

I just installed it on a new machine and it is now broken. When I ran it it says:
```
systemctl status ydns-updater.service
● ydns-updater.service - yDNS Updater
   Loaded: loaded (/etc/systemd/system/ydns-updater.service; disabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Sat 2021-01-16 22:34:08 UTC; 952ms ago
  Process: 7119 ExecStart=/root/go/bin/ydns-updater --daemon --host control.serveursdupeuple.net --user alauzon@alainlauzon.com --pass ""
 Main PID: 7119 (code=exited, status=1/FAILURE)

Jan 16 22:34:08 control systemd[1]: ydns-updater.service: Service RestartSec=100ms expired, scheduling restart.
Jan 16 22:34:08 control systemd[1]: ydns-updater.service: Scheduled restart job, restart counter is at 5.
Jan 16 22:34:08 control systemd[1]: Stopped yDNS Updater.
Jan 16 22:34:08 control systemd[1]: ydns-updater.service: Start request repeated too quickly.
Jan 16 22:34:08 control systemd[1]: ydns-updater.service: Failed with result 'exit-code'.
Jan 16 22:34:08 control systemd[1]: Failed to start yDNS Updater.
```

If I run it directly it says:
```
/root/go/bin/ydns-updater --daemon --host "myserver" --user "myemail" --pass "mypassword"
NAME:
   ydns-updater - A new cli application

USAGE:
   ydns-updater [global options] command [command options] [arguments...]

VERSION:
   dev, commit none, built at unknown

COMMANDS:
   help, h  Shows a list of commands or help for one command

GLOBAL OPTIONS:
   --base value       base url for api calls on ydns (default: "https://ydns.io/api/v1/update/")
   --host value       host to update
   --user value       username for authentication on ydns
   --pass value       password for authentication on ydns
   --daemon           enables the updater as a daemon (default: false)
   --frequency value  sleep time between updates while in daemon mode (default: 1h0m0s)
   --help, -h         show help (default: false)
   --version, -v      print the version (default: false)
FATA[0000]                                               error="Required flag \"base\" not set"
```

I tried to set the base flag to no avail:
/root/go/bin/ydns-updater --daemon --host "myserver" --base="https://ydns.io/api/v1/update/" --user "myemail" --pass "mypassword"
INFO[0000] updating record                               host=myserver
failed to perform request due to invalid input parameters

**Comment by wyattjoh** [comment]
Hey @alauzon! Thanks for the report!

I've addressed this bug in v3.0.1, you should be able to pull down the code again (if you're building from source) with:

```sh
go get github.com/wyattjoh/ydns-updater
```

Let me know if this addressed your issue and I can close this issue!

**Comment by alauzon** [comment]
Yes it works!
Thank you very much for the fix!
I had to remove the files un der the go directory before doing 
`go get github.com/wyattjoh/ydns-updater`
and then it worked!
